gangsta_chess/logic/board/chess_board.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+from gangsta_chess.logic.piece.piece import Piece
+from gangsta_chess.logic.board.board import Board
+from gangsta_chess.logic.piece.pawn import Pawn
+from gangsta_chess.logic.piece.bishop import Bishop
+from gangsta_chess.logic.piece.king import King
+from gangsta_chess.logic.piece.knight import Knight
+from gangsta_chess.logic.piece.queen import Queen
+from gangsta_chess.logic.piece.rook import Rook
+
+
+class ChessBoard(Board):
+    def __init__(self, Pieces, shape=(8, 8)):
+        super().__init__(Pieces, shape)
+        self.back_row = np.array([Rook, Knight, Bishop, King, Queen, Bishop, Knight, Rook])
```

Comment:
the piece array could be passed in as an argument with a default value to support dependency injection but we probably wont need to change this ever so its up to you